muni_portal/settings.py

```diff
@@ -111,6 +112,15 @@
 DATABASES["default"]["CONN_MAX_AGE"] = env.int("CONN_MAX_AGE", default=60)  # noqa F405
 
 
+# https://docs.djangoproject.com/en/2.2/topics/cache/#setting-up-the-cache
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'LOCATION': 'unique-snowflake',
+    }
+}
```

Comment:
Why does this PR need caching?
But if we're adding caching, please use this approach:
https://github.com/keep-the-receipts/procurement-portal-backend/pull/49/files